src/ngAutocomplete.js

```diff
@@ -181,13 +196,6 @@ angular.module( "ngAutocomplete", [])
                                                     element.val(detailsResult.formatted_address);
 
                                                     scope.details = detailsResult;
-
-                                                    //on focusout the value reverts, need to set it again.
-                                                    var watchFocusOut = element.on('focusout', function(event) {
```

Comment:
Are we sure this should be removed - what about other cases like full address etc.?